build.sbt

```diff
@@ -16,16 +19,16 @@ assemblyOption in assembly ~= {
   _.copy(cacheUnzip = true)
 }
 
-val pureconfigVersion = "0.8.0"
+val pureconfigVersion = "0.12.1" //didn't work with 0.11.1 but works with this strangely
 
 //Define dependencies.
 libraryDependencies ++= Seq(
-  "org.scalatest"         %% "scalatest"             % "3.0.1" % "test",
-  "org.scalacheck"        %% "scalacheck"            % "1.13.4" % "test",
+  "org.scalatest"         %% "scalatest"             % "3.2.0-M1" % "test", //"3.0.1" % "test",
+  "org.scalacheck"        %% "scalacheck"            % "1.14.2" % "test", //"1.13.4" % "test",
   "com.typesafe"          % "config"                 % "1.3.1",
   "com.github.pureconfig" %% "pureconfig"            % pureconfigVersion,
   "com.github.pureconfig" %% "pureconfig-enumeratum" % pureconfigVersion,
-  "com.monovore"          %% "decline"               % "0.3.0",
+  "com.monovore"          %% "decline"               % "1.0.0", //"0.3.0",
```

Comment:
Remove commented code.

Reply:
Thanks Dan. I will clean up and push a cleaned version.